python/dgl/nn/pytorch/conv/cugraph_relgraphconv.py

```diff
@@ -214,87 +87,68 @@ def __init__(
         regularizer=None,
         num_bases=None,
         bias=True,
-        activation=None,
```

Comment:
Is this a breaking change?

Reply:
Yes. I saw the this comment from RelGraphConv:
https://github.com/dmlc/dgl/blob/76bb54044eb387e9e3009bc169e93d66aa004a74/python/dgl/nn/pytorch/conv/relgraphconv.py#L128-L129

I also prefer to apply activation outside of the conv layer.